### PR TITLE
(#8) Commit circle.yml file.

### PR DESCRIPTION
**Pull request by scottrice10**

### CODE REVIEW

File: client/index.html [hunk 1]
```diff
@@ -46,6 +53,15 @@
...
+      <script src="bower_components/angular-resource/angular-resource.js"></script>
+      <script src="bower_components/angular-cookies/angular-cookies.js"></script>
+      <script src="bower_components/angular-sanitize/angular-sanitize.js"></script>
+      <script src="bower_components/angular-bootstrap/ui-bootstrap-tpls.js"></script>
+      <script src="bower_components/lodash/dist/lodash.compat.js"></script>
+      <script src="bower_components/angular-socket-io/socket.js"></script>
+      <script src="bower_components/angular-ui-router/release/angular-ui-router.js"></script>
       <!-- endbower -->
       <script src="socket.io-client/socket.io.js"></script>
     <!-- endbuild -->
```
Comment (kxprim) [inline client/index.html:64]: Indentation seems off

**Comment by rvbsanjose** [comment]
This is done by the generator so we should fix this before merging.

**Comment by peerwit** [comment]
:+1:

**Comment by rvbsanjose** [comment]
:+1:

**Pull request merged**